coalib/bearlib/languages/definitions/C.py

```diff
@@ -19,3 +19,4 @@ class C:
         '#undef', '#ifdef', '#ifndef', '#if', '#endif', '#else', '#elif',
         '#line', '#pragma']
     special_chars = list(r'+-*/.;\,()[]{}\=<>|&^~?%!')
+    escape_quotes = '\"',
```

Comment:
We can merge this if they all change to being dictionaries which describe how to escape string delimiters

`string_delimiter_escape = {'"': '\\"'}`